queries.sql

```diff
@@ -55,4 +55,89 @@ animals
  FROM
  animals
  WHERE 
- weight_kg >=10.4 AND weight_kg <=17.3;
\ No newline at end of file
+ weight_kg >=10.4 AND weight_kg <=17.3;
+
+-- Update animals with names ending in 'mon'
+UPDATE 
+animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update animals with no species set
```

Comment:
- [ ] As mentioned by the previous reviewer your queries are not according to the requirement, here the requirement states to `UPDATE` the animal's table by setting the species column to unspecified, then roll back the change. Here you are explicitly asked to verify that the change was made and rollbacked. Please add the `SELECT` statement before and after the change is rollback.